ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -2992,6 +2996,8 @@
 				GCC_NO_COMMON_BLOCKS = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				PRODUCT_BUNDLE_IDENTIFIER = "org.reactivecocoa.$(PRODUCT_NAME:rfc1034identifier)-Tests";
+				PRODUCT_NAME = "$(PROJECT_NAME)";
```

Comment:
Or you need to set `TARGET_NAME` differently for the tests because they're getting the same name as the framework